its/src/test/java/com/sonar/it/scanner/msbuild/sonarcloud/IncrementalPRAnalysisSonarCloudTest.java

```diff
@@ -158,6 +159,8 @@ private static void runBeginStep(Path projectDir, StreamConsumer.Pipe logsConsum
       .addArgument("/o:" + SONARCLOUD_ORGANIZATION)
       .addArgument("/k:" + SONARCLOUD_PROJECT_KEY)
       .addArgument("/d:sonar.host.url=" + SONARCLOUD_URL)
+      .addArgument("/d:sonar.scanner.sonarcloudUrl=" + SONARCLOUD_URL)
+      .addArgument("/d:sonar.scanner.apiBaseUrl=" + SONARCLOUD_API_URL)
```

Comment:
These are not used right now by the scanner-cli, but they will be needed when we bump it.
They are passed through `ProcessedArgs` all the way to the end step via `SonarQubeAnalysis.xml`.
After we bump the scanner cli, we can omit `sonar.host.url`.

Reply:
Shouldn't the `sonarcloudUrl` parameter be optional even after the CLI update?

From the docs:
> For backward compatibility, if the value of sonar.host.url was set to https://sonarcloud.io by the user (with or without trailing slashes), then the scanner should consider it is connecting to SonarCloud.

Reply:
No, as we are not targeting `sonarcloud.io`, but `sc-staging`.
If we were targeting `sonarcloud.io`, you would be absolutely correct, we would not need to set anything at all.

But now we need `sonar.host.url` for the current CLI version, and after we bump it we can drop it and keep the other 2 properties for web/api URIs.